src/cow-react/modules/limitOrders/pure/RateInfo/index.tsx

```diff
@@ -31,39 +33,51 @@ const InvertIcon = styled.div`
   cursor: pointer;
 `
 
-export function RateInfo({ activeRateDisplay, className }: RateInfoProps) {
+export function InvertRateControl({ onClick }: { onClick(): void }) {
+  return (
+    <InvertIcon onClick={onClick}>
+      <RefreshCw size={12} />
+    </InvertIcon>
+  )
+}
+
+export function RateInfo({ activeRateDisplay, className, isInversed = false, noLabel = false }: RateInfoProps) {
   const { inputCurrency, outputCurrency, activeRate, activeRateFiatAmount, inversedActiveRateFiatAmount } =
     activeRateDisplay
 
-  const [isInversed, setIsInversed] = useState(false)
+  const [currentIsInversed, setIsInversed] = useState(isInversed)
```

Comment:
To avoid confusion, can you keep the property and setter names consistent?
One might think that `setIsInversed` will set `isInversed` property